### PR TITLE
Refactor Website Sleeping page

### PR DESCRIPTION
**Pull request by rapkis**
Refactored the website sleeping page according to the new designs.
Related issue: https://github.com/hostinger/design/issues/154
* Added links to 000webhost homepage, login page, signup page and hostinger.com/special/000webhost page.
* The new designs required a lot of CSS additions.

New page screenshots desktop and mobile:
image
image

* Page Audits old vs new (localhost):
image
image
Best practices is due to localhost

P.S. After the review, the code should be minimised to increase page load speed. Will minimise after, for easier review

### CODE REVIEW

File: website-sleeping.html [hunk 1]
```diff
@@ -12,44 +12,32 @@
...
-        height: 100%;
+        background-color: white;
         -webkit-font-smoothing: antialiased;
     }
 
     body {
         margin: 0;
         padding: 0;
         display: flex;
+        display: -webkit-flex;
```
Comment (LmZeD) [inline website-sleeping.html]: Mozilla flex?

File: website-sleeping.html [hunk 2]
```diff
@@ -61,119 +49,520 @@
...
+        max-width: 100%;
+        height: auto;
+    }
+
+    .container {
+        padding: 0 5%;
+        width: 90%;
+    }
+
+    .flex {
```
Comment (LmZeD) [inline website-sleeping.html]: prefixers

File: website-sleeping.html [hunk 3]
```diff
@@ -61,119 +49,520 @@
...
+        cursor: pointer;
+        transition: box-shadow 0.2s;
+    }
+    /* Overlay */
+    .button:before {
+        content: "";
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
```
Comment (laurynaszilinskas) [inline website-sleeping.html]: Maybe just top/left ?

File: website-sleeping.html [hunk 4]
```diff
@@ -61,119 +49,520 @@
...
+    .button:hover:focus:before {
+        opacity: 0.3;
     }
 
-    .hk-logo, .app-icon {
-        fill: #DBE1EC;
+    .button:active:after {
+        opacity: 0.32;
+        transform: translate(-50%, -50%) scale(0);
+        transition: transform 0s;
```
Comment (laurynaszilinskas) [inline website-sleeping.html]: 0s is the same as no transition?

File: website-sleeping.html [hunk 5]
```diff
@@ -61,119 +49,520 @@
...
+        content: "";
+        position: absolute;
+        left: 50%;
+        top: 50%;
+        border-radius: 50%;
+        width: 32px; /* Safari */
+        height: 32px; /* Safari */
+        background-color: rgba(255, 92, 98, 0.3);
+        opacity: 0;
+        transform: translate(-50%, -50%) scale(1);
```
Comment (laurynaszilinskas) [inline website-sleeping.html]: Scale(1) is default scale or is it used for transition purposes?

File: website-sleeping.html [hunk 6]
```diff
@@ -61,119 +49,520 @@
...
+            <div class="flex flex-column card bg-white">
+                <div class="flex justify-content-center card__image-holder">
+                    <img src="https://cdn.000webhost.com/000webhost/000webhost-pages/corgi-make-a-website.svg" class="full-width" alt="">
+                </div>
+                <h3 class="heading-3 text-bold text-center text-charcoal-grey mb-10">Create Your Own Website</h3>
+                <p class="paragraph text-battleship-grey text-center mb-20">
+                    If you're just a visitor of the page - try creating your own website for free and get it up and running in just a matter of seconds. Your dream website is waiting for you!
+                </p>
+                <div class="flex justify-content-center full-height">
+                    <a class="button button--primary card__button align-self-end" href="https://www.000webhost.com/free-website-sign-up?utm_source=000webhost&utm_medium=frontend&utm_campaign=website-sleeping">Get Started</a>
```
[the remark this answers is not in the document]
Comment (rapkis) [reply website-sleeping.html]: We simply redirect

File: website-sleeping.html (hunk 4 above)
Comment (rapkis) [reply website-sleeping.html]: This is required for transition

File: website-sleeping.html [hunk 7]
```diff
@@ -61,119 +54,602 @@
...
+            <a class="button button--secondary main-ctas__button" href="https://www.000webhost.com/?utm_source=000webhost&utm_medium=frontend&utm_campaign=website-sleeping">Homepage</a>
+        </div>
+        <div class="scroll-holder--mobile">
+            <div class='icon-scroll'></div>
+            <span class="text-coral-pink text-bold text-scroll">SCROLL DOWN</span>
+        </div>
+        <div class="image-holder flex justify-content-center">
+            <img src="https://cdn.000webhost.com/000webhost/000webhost-pages/corgi-sleeping-on-a-table.svg" class="main-image" alt="Dog sleeping on a laptop">
+            <div class="scroll-holder">
+                <div class='icon-scroll'></div>
```
Comment (laurynaszilinskas) [inline website-sleeping.html]: Single qoutes

File: website-sleeping.html [hunk 8]
```diff
@@ -61,119 +54,602 @@
...
+        <div class="flex justify-content-evenly main-ctas">
+            <a class="button button--primary mr-30 main-ctas__button" href="https://www.000webhost.com/cpanel-login?utm_source=000&utm_medium=website-sleeping&utm_campaign=pages">Wake website from sleep</a>
+            <a class="button button--secondary main-ctas__button" href="https://www.000webhost.com/?utm_source=000webhost&utm_medium=frontend&utm_campaign=website-sleeping">Visit 000Webhost Homepage</a>
+        </div>
+        <div class="justify-content-between main-ctas--mobile full-width">
+            <a class="button button--primary main-ctas__button main-ctas__button--spaced" href="https://www.000webhost.com/cpanel-login?utm_source=000&utm_medium=website-sleeping&utm_campaign=pages">Wake your site</a>
+            <a class="button button--secondary main-ctas__button" href="https://www.000webhost.com/?utm_source=000webhost&utm_medium=frontend&utm_campaign=website-sleeping">Homepage</a>
+        </div>
+        <div class="scroll-holder--mobile">
+            <div class='icon-scroll'></div>
```
Comment (laurynaszilinskas) [inline website-sleeping.html]: single quotes

**Pull request merged**